### PR TITLE
 #SymfonyConHackDay2017 add changes to composer.json for Symfony4 support

### PR DESCRIPTION
**Pull request by llaakkkk**
| Q | A
| --- | ---
| Branch? | 2.0 
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? |no
| Tests pass? | no
| Fixed tickets | 
| License | MIT
| Doc PR | 

On SymfonyCon Cluj 2017 we add changes to bundles for Symfony 4 support. Also, added changes to composer.json ^4.0 and to .trevis.yml for tests.

### CODE REVIEW

**Comment by llaakkkk** [comment]
#SymfonyConHackday2017

**Comment by lsmith77** [comment]
looks good .. will wait until https://github.com/php-enqueue/enqueue-bundle/pull/1 is merged

**Comment by michaelperrin** [comment]
@lsmith77 It seems that it has been merged now (the PR being this one: https://github.com/php-enqueue/enqueue-dev/pull/269)

**Comment by llaakkkk** [comment]
@lsmith77 I'll try to check today

File: composer.json [hunk 1]
```diff
@@ -36,19 +36,18 @@
         "doctrine/cache": "~1.1",
         "doctrine/orm": "~2.3",
         "enqueue/enqueue-bundle": "^0.7",
```
Comment (lsmith77) [inline composer.json]: IIRC for versions below 1.0, we need to explicitly allow minor versions .. so I think this needs to be “^0.7|^0.8” in order to be able to pull in the 0.89 version which provides Symfony compatibility

Comment (lsmith77) [reply composer.json]: fixed via https://github.com/liip/LiipImagineBundle/pull/1013/commits/c5e26d6ac908fc96f801ca070415b648b66e694f

**Pull request merged**